### PR TITLE
gives ComicInfo.xml files priority over noxml files if both are at th…

### PR DESCRIPTION
**Pull request by Shamicen**
This gives `ComicInfo` files priority over `noxml` files. If both a `.noxml` files and a `ComicInfo.xml` file are inside the chapter root Tachiyomi will now use the ComicInfo file to set Metadata from it.

### CODE REVIEW

File: app/src/main/java/eu/kanade/tachiyomi/source/LocalSource.kt [hunk 1]
```diff
@@ -151,7 +151,7 @@ class LocalSource(
         try {
             val mangaDirFiles = getMangaDirsFiles(manga.url, baseDirsFile).toList()
             val comicInfoMetadata = mangaDirFiles
-                .firstOrNull { it.name == COMIC_INFO_FILE || it.name == ".noxml" }
+                .lastOrNull { it.name == COMIC_INFO_FILE || it.name == ".noxml" }
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/source/LocalSource.kt:154]: Considering we already fetch all the files as `mangaDirFiles`, we should just check for the 2 files individually. If a `ComicInfo.xml` file exists, we should delete the `.noxml` file.

**Comment by arkon** [comment]
Does this fix #8112?

**Comment by Shamicen** [comment]
> Does this fix #8112?

No

File: app/src/main/java/eu/kanade/tachiyomi/source/LocalSource.kt (hunk 1 above)
Comment (Shamicen) [reply app/src/main/java/eu/kanade/tachiyomi/source/LocalSource.kt:154]: OK will do

Comment (Shamicen) [reply app/src/main/java/eu/kanade/tachiyomi/source/LocalSource.kt]: I implemented it in the new commit

**Pull request merged**